### DISCUSSION TITLE
Lmax in NequIP vs. ACE B2 Descriptor

### DISCUSSION THREAD

**Discussion by potus28** [mir-group/nequip#394] [General]
Hi everyone,

I'm working on training force fields for liquid argon using NequIP, Allegro, and FLARE. From what I understand, lmax in both FLARE if you are using the ACE B2 descriptor and NequIP / Allegro controls the maximum spherical harmonics degree that is included in either the B2 descriptor or used to build the equivariant decsriptor for the neural networks. I'm doing hyperparameter scans of FLARE and NequIP, and what I was surprised by is that for the B2 descriptor in FLARE, setting lmax = 5 gave the best results as opposed to NequIP where lmax = 3 was the best choice. I know what makes NequIP / Allegro unique is that it takes advantage of equivariant descriptors, whereas ACE, to my understanding, is a invariant descriptor only. Is that why for NequIP I don't need a larger lmax compared to the B2 descriptor or is there something else going on? Thanks!

Sincerely,
Woody
